src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <h1> FORM </h1>
+      <p> Будь ласка заповність форму</p>
+      <script type="text/javascript" src="./main.js"></script>
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+     Surname:
```

Comment:
the tabulation should be fixed
```suggestion
        <div class="form-field">
          Surname:
```